rhobs/alerting/data_plane/integration_service_response.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: integration-service-response-alerting-rules
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: integration-service-response-exceeded
+    interval: 1m
+    rules:
+    - alert: IntegrationServiceResponse
+      expr:  sum(increase(integration_svc_response_seconds_bucket[$__interval])) by (le)
+     
```

Comment:
this empty line shouldn't be here imho, and also at the end of the should be a newline separator